src/metabase/actions/args.clj

```diff
@@ -0,0 +1,179 @@
+(ns ^{:instrument/always true} metabase.actions.args
+  (:require
+   [metabase.legacy-mbql.normalize :as mbql.normalize]
+   [metabase.util :as u]
+   [metabase.util.log :as log]
+   [metabase.util.malli.registry :as mr]
+   [toucan2.core :as t2]))
+
+(defmulti action-arg-map-schema
```

Comment:
we can remove this if we put a output schema for all `normalize-action-arg-map` defmethod because it's only used to validate output of arg normalization., but I think Chris might need this defmulti for his next task so will leave it here for now.